File: FutronicSDK/Examples/Net/Vs2013/WorkedEx.VB/EnrollmentName.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms

Namespace Futronic.SDK.WorkedEx
	Public Partial Class EnrollmentName
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Public ReadOnly Property UserName() As String
			Get
				Return txtUserName.Text
			End Get
		End Property
	End Class
End Namespace

File: FutronicSDK/Examples/Net/Vs2010/WorkedEx/EnrollmentName.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace Futronic.SDK.WorkedEx
{
    public partial class EnrollmentName : Form
    {
        public EnrollmentName()
        {
            InitializeComponent();
        }

        public String UserName
        {
            get
            {
                return txtUserName.Text;
            }
        }
    }
}